repos/spack_repo/builtin/packages/pandoc/package.py

```diff
@@ -50,22 +57,43 @@ class Pandoc(Package):
             url="https://github.com/jgm/pandoc/releases/download/2.7.3/pandoc-2.7.3-linux.tar.gz",
         )
 
+    elif platform.system() == "Darwin" and platform.machine() == "aarch64":
+        url = "https://github.com/jgm/pandoc/releases/download/3.7.0.2/pandoc-3.7.0.2-arm64-macOS.zip"
+
+        version(
+            "3.7.0.2", sha256="66a579bd8aae83de0bbeba43900953b075a6a3caaa7d1bfc19173e8f95d2ea17"
```

Comment:
When I run this on my Mac (`spack arch` yields `darwin-sequoia-m1`), I get the sha256 on line 71, not this one.

Reply:
I suspect the check at line 60 should be of `arm64` and not `aarch64`.

Reply:
🤔 But with that change the older versions (lines 70-92) aren't available.